Cargo.toml

```diff
@@ -16,15 +16,15 @@ readme = "README.md"
 serde = { version = "1.0.183", features = ["derive"], optional = true}
 thiserror = "1.0"
 
+[build-dependencies]
+cc = "1.0.73"
+
```

Comment:
`cc` is not used on Windows, so perhaps its better to scope this dependency to Unix based OS?